### DISCUSSION TITLE
Help

### DISCUSSION THREAD

**Discussion by EmeraIdKing** [eritislami/evobot#860] [Q&A]
Im trying to set up so the bot doesn't crash after 1 hour, i found uptimerobot to be working just fine for other bots, but the problem is i need a custom domain thingy? (webserver?) for that, and i dont see this one having one on replit. 
help!

**Comment by Takiyo0** [comment]
if you're using replit and want to use uptimerobot, you just need to create 1 file `server.js` and install 1 module `express`.
```js
// server.js file
const express = require('express');
const app = express();
require('./index.js');

app.get('/', (req, res) => {
res.sendStatus(200)
})

app.listen(process.env.PORT || 3000, () => console.log(`Now listening to port ${process.env.PORT || 3000}`))
```
and start your bot with `node server.js` and not `node index.js`
